index.js

```diff
@@ -207,6 +208,30 @@ class Ora {
 		this.updateLineCount();
 	}
 
+	get isEnabled() {
+		return this._isEnabled && !this.isSilent;
+	}
+
```

Comment:
Converted `isEnabled` to a getter and forcing it to `false` when `isSilent` is `true`.  This reduced the footprint of the PR and prevented double checks of both flags.